viz-lib/src/visualizations/sankey/initSankey.js

```diff
@@ -109,10 +123,33 @@ function spreadNodes(height, data) {
 
 function isDataValid(data) {
   // data should contain column named 'value', otherwise no reason to render anything at all
-  return data && !!find(data.columns, c => c.name === "value");
+  if (!data || !find(data.columns, c => c.name === "value")) {
+    return false;
+  }
+  // prepareData will have coerced any invalid data into NaN, which is accumulated by the reducers below
+  const doAllRowsContainValidData = !isNaN(
+    reduce(data.rows, (finalAcc, row) => finalAcc * reduce(row, (acc, col) => col * acc, 0), 0)
+  );
+
+  return doAllRowsContainValidData;
+}
+
+// will mutate data into valid numbers
+function prepareData(data) {
+  const { rows } = data;
+  const result = map(rows, row => {
+    const newRow = {};
+    each(row, (value, key) => {
+      newRow[key] = +value;
+    });
+    return newRow;
+  });
+  data.rows = result;
```

Comment:
Add `mapValues` import from `lodash` and then:

```suggestion
  data.rows = map(data.rows, row => mapValues(row, v => parseFloat(v)));
```

Also, quick question: is it safe to mutate `data` object here?